pkg/network/ebpf/c/tracer-telemetry.h

```diff
@@ -47,7 +47,7 @@ static __always_inline void increment_telemetry_count(enum telemetry_counter cou
 static __always_inline void sockaddr_to_addr(struct sockaddr * sa, u64 * addr_h, u64 * addr_l, u16 * port) {
     if (!sa) return;
 
-    u16 family;
+    u16 family = AF_INET;
```

Comment:
Is there any reason this is set to AF_INET ? Since the value immediately gets overwritten it shouldn't matter. In most cases like this we initialize to zero.